packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -453,6 +459,17 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     const { triggerButton } = this.props
     const content = this.getSelectedItemAsString(this.state.value)
 
+    const toggleButtonProps = getToggleButtonProps({
```

Comment:
is `toggle` and `trigger` button the same thing? Would suggest to maintain naming consistency in this case

Reply:
`getToggleButtonProps` as a method comes from Downshift, and we are using this method on the `triggerButton` - main area of the single selection Dropdown, and the `toggleIndicator`, the arrow on the right side of the `Dropdown`. Can you please let me know what should be renamed in this case?

Reply:
given that method is named `renderTriggerButton`, as well as that props contained in _scoped_ variable `toggleButtonProps` are applied to trigger button only in the context of this method, I would rename it as `triggerButtonProps` - yes, we have `getToggleButtonProps` as method from `Downshift`, but in the context of our method, where everything is scoped to this method, I would expect to see everything related to `triggerButton`, accordingly

Reply:
Got it, will rename the const to triggerButtonProps.